plugin/main/src/kotlinx/benchmark/gradle/Utils.kt

```diff
@@ -256,9 +256,5 @@ internal val Gradle.isConfigurationCacheAvailable
     } ?: false
 
 internal fun Project.getSystemProperty(key: String): String? {
-    return if (gradle.isConfigurationCacheAvailable) {
-        providers.systemProperty(key).forUseAtConfigurationTime().orNull
```

Comment:
According to the migration guide:
> The Provider based APIs are still the recommended way to connect external values to task inputs for maximum configuration cache reuse.

Should `providers.systemProperty(key).orNull` be used instead of `System.getProperty(key)`?